client/plots/sampleView.js

```diff
@@ -129,11 +129,20 @@ class SampleView extends MassDict {
 
 	getTermValue(term) {
 		let value = this.sampleData[term.id]?.value
-
 		if (value == null || value == undefined) return null
 		if (term.type == 'float' || term.type == 'integer')
 			return value % 1 == 0 ? value.toString() : value.toFixed(2).toString()
 		if (term.type == 'categorical') return term.values[value].label || term.values[value].key
+		if (term.type == 'condition') {
+			let [years, status] = value.split(' ')
+			status = term.values[status].label || term.values[status].key
+			return `${status} after ${Number(years).toFixed(1)} years`
```

Comment:
please replace wording as `Max grade: xx, time-to-event: yy years`

Reply:
Updated now, survival terms are ok?

Reply:
hope that's okay. we can always change later. thanks